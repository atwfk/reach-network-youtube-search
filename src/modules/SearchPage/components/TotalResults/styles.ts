import styled from "styled-components";

export const StyledSearchResults = styled.h3`
  font-weight: 400;
`;
