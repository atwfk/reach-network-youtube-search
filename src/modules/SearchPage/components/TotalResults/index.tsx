import React from "react";
import type { FC, ReactElement } from "react";
import { formatNumberWithCommas } from "../../../shared/logic/transformIntNum/transformIntNum";
import { StyledSearchResults } from "./styles";

const TotalResults: FC<{ totalResults: number }> = ({ totalResults }): ReactElement => {
  return (
    <StyledSearchResults>
      About {formatNumberWithCommas(totalResults)} filtered results
    </StyledSearchResults>
  );
};

export default TotalResults;
