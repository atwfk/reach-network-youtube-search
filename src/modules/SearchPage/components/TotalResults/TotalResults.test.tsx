import { cleanup } from "@testing-library/react";
import "jest-styled-components";
import renderer from "react-test-renderer";
import TotalResults from ".";

afterEach(cleanup);

describe("TotalResults Component snapshots", () => {
  it("Should render simple TotalResults component", () => {
    const tree = renderer.create(<TotalResults totalResults={131909} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
