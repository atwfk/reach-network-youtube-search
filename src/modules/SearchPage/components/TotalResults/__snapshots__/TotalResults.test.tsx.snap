// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TotalResults Component snapshots Should render simple TotalResults component 1`] = `
.c0 {
  font-weight: 400;
}

<h3
  className="c0"
>
  About 
  131,909
   filtered results
</h3>
`;
